doc/nrf/ug_dev_model.rst

```diff
@@ -120,6 +120,12 @@ The Git tags follow the `Semantic Versioning 2.0.0`_ format and they correspond
 
 Where X, Y, and Z are the major, minor, and patch version respectively and, optionally, a release candidate postfix ``-rcN`` is attached if the tag identifies a candidate instead of the actual release.
 
+Additionally, in between releases zero or more development tags can be present. Those come in the form::
+
+  vX.Y.99-devN
+
+Where X and Y are the major and minor version numbers respectively and N is the development tag number.
+
```

Comment:
Suggest to integrate this information into the information above (from line 117):

```suggestion
The Git tags follow the `Semantic Versioning 2.0.0`_ format as follows::

  vX.Y.Z(-rcN|-devN)

X, Y, and Z are the major, minor, and patch version, respectively.
Tags without a suffix correspond to official releases tested and signed by Nordic Semiconductor engineers. 
A release candidate suffix ``-rcN`` is attached if the tag identifies a candidate instead of the actual release.
In between releases, there might be development tags.
These are identified by a ``-devN`` suffix.